kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/servlet/FileServlet.java

```diff
@@ -858,6 +858,7 @@ class UploadRequest extends ServletFileUpload {
     public UploadRequest(DiskFileItemFactory diskFileItemFactory) {
         super(diskFileItemFactory);
         setSizeMax(FileServlet.getFileUploadSizeMax());
+        setFileCountMax(2L);
```

Comment:
Why is this needed? And why are max. 2 files allowed per request?

Reply:
I have tried with 1 and it does not allow to upload. I have not checked in the code but maybe the limit is exclusive